modules/userId/eids.md

```diff
@@ -182,13 +190,6 @@ userIdAsEids = [
             atype: 3
         }]
     },
-    {
-        source: 'akamai.com',
```

Comment:
This akamai block should not be removed from eids.md. All other files are fine

Reply:
I believe it was a mistake, I added again.